dev/README.md

```diff
@@ -50,6 +50,31 @@ In a separate terminal session/pane/window:
     sudo -i
     dvm quickstart oc_erchef
 
+To use your running Chef Server through standard commands such as knife,
+you'll need to create an organisation and a user, and then create a
+knife config on your workstation.
+
+    vagrant ssh
+    sudo -i
+    # create a user to access chef with
+    chef-server-ctl user-create -f /tmp/admin.pem admin Admin User admin@example.com password 
+    # create an organization
+    chef-server-ctl org-create -f /tmp/test-validator.pem test Test
```

Comment:
You can add `-a admin` and the user will automatically be added to the org so you don't need the step below.